tools/pot/setup.py

```diff
@@ -65,15 +65,15 @@ def get_version():
     sys.argv.remove('--install-dev-extras')
 
 INSTALL_REQUIRES = [
+    "numpy>=1.16.6,<=1.23.1; python_version < '3.11'",
+    "numpy>=1.23.2; python_version >= '3.11'",
```

Comment:
It will not be aligned with other requirements files:(

Reply:
Does it cause any issues or it is just about misalighment?

Reply:
Numpy packages for Python 3.11 are available starting from 1.23.2. You can check yourself here: https://numpy.org/devdocs/release/1.23.2-notes.html

Reply:
Agreed that it is not relevant anymore since the numpy upper bound has been removed from the package requirements.